R/birdlife.R

```diff
@@ -25,3 +25,46 @@ birdlife_habitat = function(id){
   out = cbind(id, tables[[habitat_table_number]])
   out[-nrow(out), ] # Drop last row (altitude)
 }
+
+
```

Comment:
@davharris don't we need an `#' @export` here - and a `#' @rdname birdlife_habitat`, or if you want to give it a separate man file, then that stuff